### PR TITLE
docs(configure-redis-using-configmap): update for clarity

### PR DESCRIPTION
**Pull request by jai**
Fixes https://github.com/kubernetes/website/issues/25638

Signed-off-by: Jai Govindani <jai@honestbank.com>

### CODE REVIEW

**Comment by jai** [comment]
I tried to follow the style guide in terms of separating commands and output - it leads to a bit of verbosity in some areas, especially the part where we walk the reader through checking the Redis config, but it's not that bad I guess. The previous version combined commands/output - I'm not sure if this style guide rule is definitive or is applied on a case-by-case basis?

**Comment by jai** [comment]
bump - anyone know if there's anything else I need to do to get this merged?

**Comment by celestehorgan** [comment]
Hi @jai, 

Thanks for doing this! Tutorials don't always have a clear owner in Kubernetes-land which is why this is taking some time to review :) 

I'll test it shortly.

**Comment by celestehorgan** [comment]
Preview link: https://deploy-preview-25712--kubernetes-io-master-staging.netlify.app/docs/tutorials/configuration/configure-redis-using-configmap/

File: content/en/docs/tutorials/configuration/configure-redis-using-configmap.md [hunk 1]
```diff
@@ -38,82 +36,218 @@ This page provides a real world example of how to configure Redis using a Config
...
 
-Add the pod resource config to the `kustomization.yaml`:
+Apply the ConfigMap created above, along with a Redis pod manifest:
+
+```shell
+kubectl apply -f redis-config.yaml
+kubectl apply -f https://raw.githubusercontent.com/kubernetes/website/master/content/en/examples/pods/config/redis-pod.yaml
+```
+
+Examine the contents of the Redis pod manifest and note the follwing:
```
Comment (kbhawkey) [inline content/en/docs/tutorials/configuration/configure-redis-using-configmap.md:61]: nit: spelling, s/follwing/following/

File: content/en/docs/tutorials/configuration/configure-redis-using-configmap.md [hunk 2]
```diff
@@ -38,82 +36,218 @@ This page provides a real world example of how to configure Redis using a Config
...
 ```shell
-curl -OL https://k8s.io/examples/pods/config/redis-config
-
-cat <<EOF >./kustomization.yaml
-configMapGenerator:
-- name: example-redis-config
-  files:
-  - redis-config
+cat <<EOF >./example-redis-config.yaml
+apiVersion: v1
```
Comment (kbhawkey) [inline content/en/docs/tutorials/configuration/configure-redis-using-configmap.md:45]: Hi @jai . Thanks for creating this pull request.
I ran the updated tutorial. The commands ran successfully except for line 57,
`kubectl apply -f redis-config.yaml`. Based upon line 44, `cat <<EOF >./example-redis-config.yaml`,
I would expect `kubectl apply -f example-redis-config.yaml`?

Comment (kbhawkey) [reply content/en/docs/tutorials/configuration/configure-redis-using-configmap.md:45]: Hi @jai.
Just checking if you have time to address the comments or need more information?

Comment (jai) [reply content/en/docs/tutorials/configuration/configure-redis-using-configmap.md:45]: On it as we speak 🚀

Comment (jai) [reply content/en/docs/tutorials/configuration/configure-redis-using-configmap.md:45]: fixed

File: content/en/docs/tutorials/configuration/configure-redis-using-configmap.md (hunk 1 above)
Comment (jai) [reply content/en/docs/tutorials/configuration/configure-redis-using-configmap.md]: fixed

**Comment by kbhawkey** [comment]
Thanks @jai .
/lgtm
/label tide/merge-method-squash

**Pull request merged**